packages/parallax/src/index.tsx

```diff
@@ -46,40 +48,58 @@ export interface ParallaxLayerProps extends ViewProps {
   offset?: number
   /** Shifts the layer in accordance to its offset, values can be positive or negative */
   speed?: number
+  /** Layer will be sticky between these two offsets, all other props are ignored */
+  sticky?: { from: number; to: number }
 }
 
 export const ParallaxLayer = React.memo(
   React.forwardRef<IParallaxLayer, ParallaxLayerProps>(
-    ({ horizontal, factor = 1, offset = 0, speed = 0, ...rest }, ref) => {
+    (
+      { horizontal, factor = 1, offset = 0, speed = 0, sticky, ...rest },
```

Comment:
Is there a reason we're not destructing things like `children` when we're calling them from before?